applications/openshift/networking/default_ingress_ca_replaced/tests/ocp4/e2e.yml

```diff
@@ -1,3 +1,2 @@
 ---
-default_result: FAIL
-result_after_remediation: PASS
```

Comment:
Discussed this with @Vincent056 and @mrogers950 and it sounds like another potential option is to supply a valid ConfigMap with a legitimate CA, instead of a fake one for testing that fails later on.

This would allow us to keep the remediation in place and not clean it up later.